api-update/antivirus/src/test/scala/uk/gov/nationalarchives/api/update/antivirus/AntivirusAntivirusUpdateTest.scala

```diff
@@ -0,0 +1,101 @@
+package uk.gov.nationalarchives.api.update.antivirus
+
+import java.net.URI
+
+import com.github.tomakehurst.wiremock.client.WireMock.{equalTo, equalToJson, postRequestedFor, urlEqualTo}
+import org.scalatest.matchers.should.Matchers._
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.sqs.SqsClient
+import software.amazon.awssdk.services.sqs.model.{ReceiveMessageRequest, SendMessageRequest}
+import uk.gov.nationalarchives.api.update.common.utils.AWSInputs._
+import uk.gov.nationalarchives.api.update.common.utils.ExternalServicesTest
+
+import scala.io.Source.fromResource
+import scala.util.Try
+
+class AntivirusAntivirusUpdateTest extends ExternalServicesTest {
```

Comment:
Copy-paste error? AntivirusAntivirus.

Reply:
Antivirus Antivirus, so good they named it twice :smile: